jwst/extract_2d/grisms.py

```diff
@@ -467,9 +465,11 @@ def extract_grism_objects(input_model,
                 new_slit.meta.wcsinfo.spectral_order = order
                 new_slit.meta.wcsinfo.dispersion_direction = \
                     input_model.meta.wcsinfo.dispersion_direction
+                new_slit.meta.wcsinfo.specsys = input_model.meta.wcsinfo.specsys
+                new_slit.meta.coordinates = input_model.meta.coordinates
                 new_slit.meta.wcs = subwcs
                 if compute_wavelength:
-                    log.debug("Computing wavelengths (this takes a while ...)")
+                    log.debug("Computing wavelengths")
```

Comment:
So, it doesn't take a while anymore, or you just don't want to publicize the fact that it does?  ;-)

Reply:
The transforms were reworked a while back to evaluate on arrays instead of individual pixels but I missed this comment a the time.